api-mobile/src/paths/context/databc/{wfs}.ts

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+import axios from 'axios';
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { ALL_ROLES } from '../../../constants/misc';
+import { getLogger } from '../../../utils/logger';
+
+const defaultLog = getLogger('activity');
+
+export const GET: Operation = [getDataBC()];
+
+GET.apiDoc = {
+  description: 'Fetches elevation for a single point',
+  tags: ['activity','databc'],
+  security: [
+    {
+      Bearer: ALL_ROLES
+    }
+  ],
+  parameters: [
+    {
+      in: 'query',
+      name: 'lon',
+      required: true
+    },
+    {
+      in: 'query',
+      name: 'lat',
+      required: true
+    },
+    {
+      in: 'path',
+      name: 'wfs',
+      required: true
+    }
+  ],
+  responses: {
+    200: {
+      description: 'GeoJSON feature from DataBC Layer',
+      content: {
+        'application/json': {
+          schema: {
+            type: 'object',
+            properties: {
+            }
+          }
+        }
+      }
+    },
+    401: {
+      $ref: '#/components/responses/401'
+    },
+    503: {
+      $ref: '#/components/responses/503'
+    },
+    default: {
+      $ref: '#/components/responses/default'
+    }
+  }
+};
+
+
+/**
+ * Fetches all activity records based on request search filter criteria.
```

Comment:
I'm guilty of this too, but can you update this comment?

Reply:
Ah yes... Nice catch @micheal-w-wells .

Reply:
Also found some incorrect stuff in the OpenAPI def.